### ISSUE TITLE
Inventory not showing as back order

### ISSUE THREAD

**Issue by naurojr** [naurojr/anthology#5]
Guys, I am  training our dealers to use the portal. Client sent over the order on the portal and paid for it. As Angel tried to set up shipping she noticed in the system we only had 5 boxes not 7 like the client ordered. I just tried to order
10 boxes in the portal and it did not come up as a backorder. Can you please advise. Should we do a physical count in our warehouse. This is the second time this client had used the portal and had a issue. Let me know thanks

Bill Stagner
Sales Representative
3009 NW 75th Ave
Miami, FL 33122,USA.
Phone: 305-873-6999 Ext 303 | Fax: 305-372-9683  | Cell:954-707-0087?
Bill.S@anthologytile.com|www.anthologytile.com | www.instagram.com/anthologytile
CONFIDENTIALITY NOTICE: This email is confidential and may be privileged. If you are not the intended recipient please notify the sender immediately, and please delete it; you should not copy it or use it for any purpose or disclose its contents to any other person.

**Comment by naurojr** [comment]
After investigating we saw that item inventory number on the portal was marked as 90 boxes. That is the reason why the order or Bill's test was not flagged as a backorder. 

I've called Barbie to check the inventory history of the item and so far we're never had 90 boxes of this item. 

Syspro integration may be pushing the wrong information, or not calculating the info accurately. 

Also, it may be pushing PCS instead of boxes.
